frontend/src/App.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import './App.css';
+import Form from './components/Form/Form';
+import CategoryForm from './components/CategoryForm/CategoryForm';
+import ToDoList from './components/ToDoList/ToDoList';
+import { Category, ItemList } from './types/common';
+import Categories from './components/Categories/Categories';
+import { todoService } from './services/todoService';
+
+function App() {
+  const [allLists, setAllList] = useState<Category[]>([]);
```

Comment:
what's the reason for variable allLists to have type "Category[]" For me type "List[]" is better to undestand

Reply:
fixed